examples/file-streaming-examples.js

```diff
@@ -0,0 +1,45 @@
+const revai = require('../../revai-node-sdk');
+const fs = require('fs');
+
+/* Initialize your client with your audio configuration and access token */
+const audioConfig = new revai.AudioConfig(
+    /* contentType */ "audio/x-raw", 
```

Comment:
@kbridbur Looking at this usage without typescript, it makes me eager to change this constructor to accept an object parameter.

```
// signature
new AudioConfig(options)

// usage
new AudioConfig({
content_type: 'audio/x-raw',
layout: 'interleaved',
sample_rate: 16000,
format: 'S16LE',
channels: 1
})


```

Reply:
For my own curiosity, what are the primary arguments in favor of building an object like this?

Reply:
Josh I'm not quite sure I understand your question.

Andrew I've considered this and think it would be a reasonable change. My main issue being that it is a breaking change

Reply:
Josh's question I think (let me know if I'm off the mark) is what's the difference between declaring

````
func(var a, var b, var c)
{
}
```

vs

```
func(var obj)
{
}
```
where we have a container class `obj` which contains `var a, b, c` (this is for a constructor, but the same principals apply I think)

Preface by saying I do think it's case specific, but my arguments for usually grouping method inputs into one object is two fold. 

1. It makes it more clear what you're passing in (note you have to comment it w/o typescript)
In C# and typescript, we have named parameters, so this is alleviated somewhat, but not always which leads to 
2. that usually it's easier to track the stack call if/when you pass `obj` or it's properties around. This is more of an opinion of mine but explanation below.

There are 5 properties passed into the constructor of `AudioConfig`, and thankfully they are all related, so it's not too bad.

But say there was a 6th property like, `bool ShouldBufferAudio`. This isnt really related to the first 5, other than it's a part of `AudioConfig`. For this constructor, it's not a big deal since it's usually just setting properties, but if it was a method and the method is doing a wide variety of operations on the input parameters, it can get confusing real quick to trace what property came from where. By grouping them into logical groups, it makes it (once again, imo) easier to read.

Reply:
Makes sense, thanks. Basically the main use case of PODs.